### ISSUE TITLE
Road Restrictions - Set up Airflow job to download RR from CarteDBA into RDS

### ISSUE THREAD

**Issue by aharpalaniTO** [CityofToronto/bdit_data-sources#359]
Road restrictions are available here: `ROAD_DBA.RR_XML_IN` in the Oracle database CARTEDBA (internal Oracle DB located at 703 DM).

The timestamps are in some modified UNIX format, where you have to divide the number by 1000 before using the `TO_TIMESTAMP` function in Postgres

The first character in `ID` determines if it is a:  
R = road restriction (planned)  
I = incident

**Comment by radumas** [comment]
Airflow won't work because the EC2 can't access resources on the corporate network.

Would have to schedule using the Terminal Server

**Comment by aharpalaniTO** [comment]
Yeah I was thinking once we've moved to Canada East we could mimic what MOVE does

**Comment by radumas** [comment]
oh like down the road. sorry nvm byeeeeeee
